examples/framesjs-starter/app/debug/hub/v1/validateMessage/route.ts

```diff
@@ -0,0 +1,94 @@
+import { NextRequest } from "next/server";
+import {
+  Message,
+  FarcasterNetwork,
+  HubError,
+  ValidationResponse,
+} from "@farcaster/core";
+
```

Comment:
file should be called `mockValidateMessage`

Reply:
do you mean the folder? if so, it would break compatibility with the hub spec since the route would change

or do you mean something else?

Reply:
hm ok fine as is